bootstrap/pkg/kfapp/ksonnet/ksonnet.go

```diff
@@ -247,89 +246,7 @@ func (ksApp *ksApp) components() (map[string]*kfdefs.KsComponent, error) {
 	return comps, nil
 }
 
-func (ksApp *ksApp) deleteGlobalResources(config *rest.Config) error {
```

Comment:
where did deleteGlobalResources go?

Reply:
it's not used anymore - since we directly delete the cluster(deployment), these resources are gone with it.